Dockerfile

```diff
@@ -5,7 +5,8 @@ MAINTAINER LAA Crime Apply Team
 RUN apk add --update \
   postgresql-dev \
   tzdata \
-  yarn
+  yarn \
+  gcompat
 
 # Alpine does not have a glibc, and this is needed for dart-sass
```

Comment:
As mentioned on Review PR, I do believe we can now happily remove all these glibc related setup as either `gcompat` or maybe the bump to dart-sass makes it unnecessary, which is great!
I've tested in my non-M1 removing these lines and everything keeps working so far. Would be good to double check on an M1 machine too.